modules/statistics/php/NDB_Form_statistics.class.inc

```diff
@@ -1064,17 +1138,22 @@ class NDB_Form_Statistics extends NDB_Form
         $this->tpl_data['Sites']    = $sites;
 
         $this->tpl_data['reliability_completion']
-            = $this->getReliabilityData($Param_Project, $Param_Site);
+            = $this->getReliabilityData(
+                $Param_Project,
+                $suproject_query,
+                $Param_Site
+            );
     }
     /**
      * GetReliabilityData function
      *
-     * @param string $Param_Project the value of Param_Project
-     * @param string $Param_Site    the value of Param_Site
+     * @param string $Param_Project   the value of Param_Project
+     * @param string $suproject_query the value of suproject_query
```

Comment:
Yeah I left `suproject` as it is just in case you did it for some specific reason. I can go through and make these changes. And I was just following your comment style for consistency ;) But yeah I'll run through all the relevant files and clean them up.